tests/sentry/notifications/notification_action/test_group_type_notification_registry_handlers.py

```diff
@@ -56,3 +57,10 @@ def test_handle_workflow_action_no_handler(self, mock_registry_get):
             MetricAlertRegistryHandler.handle_workflow_action(
                 self.event_data, self.action, self.detector
             )
+
+    def test_handle_activity_update(self):
+        self.event_data = WorkflowEventData(event=self.activity, group=self.group)
+
+        with mock.patch.object(self.activity, "send_notification"):
+            execute_via_group_type_registry(self.event_data, self.action, self.detector)
+            self.activity.send_notification.assert_called_once_with()
```

Comment:
```suggestion
            self.activity.send_notification.assert_called_once()
```

If you're not going to pass anything to it you can just do `assert_called_once`

Reply:
🤔 -- it's a little different than that, `assert_called_once_with()` ensures that nothing is passed into `send_notification` where `assert_called_once` only asserts that it was called.

Reply:
Ohh I see 👍